routers/cart-router.js

```diff
@@ -0,0 +1,88 @@
+const express = require('express')
+const {Cart} = require('../models/cart-model')
+const CartRouter   = express.Router()
+CartRouter.route('/')
+.get(async (req , res) =>{
+  try{
+     const myCart = await Cart.find({})
+     res.json({myCart :myCart})
+  }catch(error){
+    res.json({status : false  , message : "couldn't collect the data" , errorMessaage : error.message})
```

Comment:
You should send it as `res.status(400).json({status : false , message : "couldn't collect the data" , errorMessaage : error.message})` .

Reply:
done